test/cadet_web/admin_controllers/admin_grading_controller_test.exs

```diff
@@ -49,45 +49,45 @@ defmodule CadetWeb.GradingControllerTest do
     @tag authenticate: :student
     test "unauthorized", %{conn: conn} do
       conn = get(conn, build_url())
-      assert response(conn, 401) =~ "User is not permitted to grade."
+      assert response(conn, 403) =~ "Forbidden"
     end
   end
 
   describe "GET /?group=true, student" do
     @tag authenticate: :student
     test "unauthorized", %{conn: conn} do
       conn = get(conn, build_url(), %{"group" => "true"})
-      assert response(conn, 401) =~ "User is not permitted to grade."
+      assert response(conn, 403) =~ "Forbidden"
     end
   end
 
   describe "GET /:submissionid, student" do
     @tag authenticate: :student
     test "unauthorized", %{conn: conn} do
       conn = get(conn, build_url(1))
-      assert response(conn, 401) =~ "User is not permitted to grade."
+      assert response(conn, 403) =~ "Forbidden"
     end
   end
 
   describe "POST /:submissionid/:questionid, student" do
     @tag authenticate: :student
     test "unauthorized", %{conn: conn} do
       conn = post(conn, build_url(1, 3), %{"grading" => %{}})
-      assert response(conn, 401) =~ "User is not permitted to grade."
+      assert response(conn, 403) =~ "Forbidden"
     end
 
     @tag authenticate: :student
     test "missing parameter", %{conn: conn} do
       conn = post(conn, build_url(1, 3), %{})
-      assert response(conn, 400) =~ "Missing parameter"
```

Comment:
Now we give a 403 Forbidden since a student would never be authorized to make this API call anyway.